packages/dual-channel/docs/zh-tw/mannual.md

```diff
@@ -0,0 +1,193 @@
+
+- 章節對應到**工作表（sheet）**（見圖 1-2）
+- 區塊則對應到工作表的**列（sheet row）** （見圖 1-2）
+- 區塊（列）可以包含四個元素：圖片、圖片動畫效果（選填）、標題（選填）和文章段落。
+
+簡單來說，「左右互搏」可以有多個章節（工作表），每個章節可以有多個區塊，每個區塊則由一張圖、一個小標和一段文字所組成。
+
+![圖1-2](../assets/figure-1.2.png)
+
+_圖 1-2：章節對應到工作表，區塊對應到工作表中的一列。_
+
+#### Spreadsheet 表格注意事項
+
+1. **工作表的名稱即是章節標題的名稱**
+2. **圖片動畫效果**欄：目前僅支援 **fadeIn** 和 **slideUp** 兩個效果（請注意英文字大小寫），若不想要有動畫效果，亦可在此欄留空不填。
+3. **圖片**欄：目前僅支援 html code（若不清楚如何填寫，可以複製範本裡的 html code 來改，將圖片網址置換成你要的即可）。在下一版本，會讓使用者輸入手機圖片網址和桌機圖片網址，由系統自動產生響應試圖片。
+4. **標題**欄：目前僅支援純文字。
+5. **文章段落**欄：目前支援純文字，也支援 html code。舉例而言，我們可以在文中插入連結，寫法如下：
+
+```html
+目前除了<a href="https://www.boca.gov.tw/cp-56-5078-41ac3-1.html"
+  >符合條件的旅客</a
+>外，中港澳人士全面暫緩入境；有中港澳旅遊史（含轉機）的人員若能入境台灣，需要接受14天的居家檢疫。
+```
+
+[符合條件的旅客](https://www.boca.gov.tw/cp-56-5078-41ac3-1.html) 在網頁上呈現時，會有超連結。
+
+當 Spreadsheet 資料準備好後，即可進入下一個步驟：產生鑲嵌程式碼。
+
+### 產生鑲嵌程式碼（Embed Code）
+
+請根據以下步驟產生「鑲嵌程式碼」：
+
+1. 瀏覽器開啟[左右互搏產生器](https://asia-east2-cloud-functions-268910.cloudfunctions.net/dual-channel)頁面
+2. 在第一個欄位裡（見圖 1-4），填入你的 Spreadsheet 的網址或是 Spreadsheet ID（見圖 1-3）
+3. 依序填入你想要出現在網頁上的工作表 ID（Sheet ID）（見圖 1-5）（註 1）
+4. 若不填工作表 ID，程式會自動抓取 Spreadsheet 上**所有的章節**。
+5. 點選 `BUILD` 按鈕
+6. 稍等一下，頁面會產生出「鑲嵌程式碼」
+7. 點選`一鍵複製`按鈕，即可將程式碼貼到你的網頁裡。
+
+![圖1-3](../assets/figure-1.3.png)
+
+_圖 1-3_
+
+![圖1-4](../assets/figure-1.4.png)
+
+_圖 1-4_
+
+＊註 1：點了 Spreadsheet 下方的工作表後（見圖 1-5），該 Spreadsheet 網址的最後會出現 `#gid=492967532`，`492967532`是該工作表的 ID，將此 ID 填入產生器中，即可產生該章節的內容。
+
+![圖1-5](../assets/figure-1.5.png)
+
+_圖 1-5_
+
+### 製圖注意事項
+
+#### 桌機版的圖
+
+請以寬 415px，長 453px 的長方形來設計。
+
+#### 手機版的圖
+
+請以正方形來設計，例如：長 375px、寬 375px。
+
+### 常見問題
+
+#### 1. 我可以將圖片上傳到哪裡呢？
+
+可以將圖片上傳到有提供圖片託管服務的網路空間，並取得圖片的公開網址。
+
+下面是幾個推薦用來上傳圖片的平台以及操作指引：
+
+- [Google 雲端硬碟](https://github.com/twreporter/orangutan-monorepo/tree/master/packages/scrollable-image/docs/zh-tw/google-drive-hosting.md)
+- [Dropbox](https://github.com/twreporter/orangutan-monorepo/tree/master/packages/scrollable-image/docs/zh-tw/dropbox-hosting.md)
+- imgur.com
+
+#### 2. 若是我不熟 HTML 和 CSS，我該如何使用「左右互搏」？
+
+現階段可以參考[Dual Channel 測試資料 - Google 試算表](https://docs.google.com/spreadsheets/d/1Ppisv4HTZHYMp95umgCoADNuP1PSkL-t9na-5lRIqSY/edit#gid=2143095237)來進行修改。
+我們會根據使用者的回饋進行調整，希望之後的版本可以滿足你的需求，
+為了進一步了解你的需求，請你給予我們回饋（請見下方[回饋](#回饋)）。
+
+#### 3. 若是圖片的大小或是長寬比不同於放圖的區塊，導致圖的呈現有些扭曲或是圖沒有被完整呈現，我該怎麼辦？
+
+由於我們在 Spreadsheet 中的**圖片**欄中，放入的值是 html code，因此可以透過 css 來調整。
+舉例而言，若我的**圖片**欄中的值為
+
+```html
+<picture
+  ><source type=“image/svg+xml” media=“(orientation: portrait)”
+  srcset=“https://www.twreporter.org/images/responsive/20200306072512-9fa12d4cc8.svg”
+  /><source type=“image/svg+xml” media=“(orientation: landscape)”
+  srcset=“https://www.twreporter.org/images/responsive/20200306072512-f3dcc75cb7.svg”
+  /><img alt=“”
+  src=“https://www.twreporter.org/images/responsive/20200306072512-9fa12d4cc8.svg”
+  style=“width:100%;height:100%” /></picture
+>
+```
+
+那我可以在 <img> 標籤中，加入 `object-fit:contain` 的方式，讓圖等比例縮放至區塊裡；亦可以使用 `object-fit:cover` 的方式，讓圖撐滿區塊，但圖的周圍會被自動裁掉。
```

Comment:
IE does not support `object-fit`. Since we have mentioned that this app support IE 10, maybe we need to add a reminder here